tests/jsunit/horizontal_tests.html

```diff
@@ -7,7 +7,7 @@
     <script>goog.require('goog.testing.jsunit');</script>
   </head>
   <body>
-    <script src="utils_test.js"></script>
+    <script src="test_utilities.js"></script>
     <script src="blockly_test.js"></script>
```

Comment:
Not related to this PR, but just noticed this file doesn't actually exist.

Reply:
Yeah.  I removed it in vertical_tests in the other testing PR.  I'm not sure what happened there.